netket/operator/_continuous_operator.py

```diff
@@ -95,3 +95,17 @@ def __mul__(self, other):
             return SumOperator(self, coefficients=other)
         else:
             return NotImplemented  # pragma: no cover
+
+    def __sub__(self, other):
+        if isinstance(self, ContinuousOperator) and isinstance(
+            other, ContinuousOperator
+        ):
```

Comment:
I mean, this is a continuous operator so self is a continuousoperator, isn't it?
```suggestion
        if isinstance(other, ContinuousOperator):
```